src/bun.js/module_loader.zig

```diff
@@ -2021,11 +2021,11 @@ pub const ModuleLoader = struct {
         } else if (HardcodedModule.Map.getWithEql(specifier, bun.String.eqlComptime)) |hardcoded| {
             switch (hardcoded) {
                 .@"bun:main" => {
-                    defer {
-                        if (jsc_vm.worker) |worker| {
-                            worker.queueInitialTask();
-                        }
-                    }
+                    // defer {
```

Comment:
is this intentional?

Reply:
worker.queueInitialTask was removing an eventloop ref way too early. since the ref was now based on the worker's event loop, the initial task to unref isn't needed i think. did not mean to leave as commented code.

Reply:
code here was removed instead of commented